tempto-examples/src/main/resources/sql-tests/datasets/sample_jdbc_table.ddl

```diff
@@ -1,5 +1,5 @@
 -- type: jdbc
 CREATE TABLE %NAME% (
-  id INT,
+  id BIGINT,
```

Comment:
This table definition was used for psql tables and tests were returning INT. While INT in Presto is not yet supported, at least it was not in 0.143 which was used for tempto testing.